compiler/rustc_ty_utils/src/instance.rs

```diff
@@ -44,7 +45,7 @@ fn inner_resolve_instance<'tcx>(
 ) -> Result<Option<Instance<'tcx>>, ErrorReported> {
     let (param_env, (def, substs)) = key.into_parts();
 
-    debug!("resolve(def={:?}, substs={:?})", def.did, substs);
+    debug!("inner_resolve_instance: key={:?}", key);
```

Comment:
ok, this one is pre-existing, but please chnage it to `#[instrument]` while you are at it